main/guides/coreeval/permissions.md

```diff
@@ -0,0 +1,89 @@
+# Coding Permissions
+
+Write a json file with the permissions that the proposal will need. For example, [here](https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/test/psm/gov-add-psm-permit.json) is the proposal Agoric created for the PSM contract:
```

Comment:
```suggestion
Write a json file with the permissions that the proposal will need. For example, [gov-add-psm-permit.json](https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/test/psm/gov-add-psm-permit.json) is the proposal Agoric created for the PSM contract:
```

Please let's keep accessibility in mind and use [meaningful link text](https://www.w3.org/WAI/tips/writing/#make-link-text-meaningful), avoiding "here" links.

Reply:
Fixed.